coordinator/tests/services/translation/request/agent-unattended.spec.ts

```diff
@@ -88,3 +90,25 @@ describe("convertOrganization", () => {
   })
 })
 
+describe("createAuthorForWithdraw", () => {
+  test("Creates an AuthorPersonSDS with correct values.", () => {
```

Comment:
worth checking the other field here too? (think theres two that get populated in withdraw author)